judge/models/tests/test_problem.py

```diff
@@ -298,9 +298,6 @@ def setUpTestData(self):
             authors=('normal',),
         )
 
-    def test_unpublished_solution(self):
```

Comment:
why?

Reply:
That was my mistake. I've recovered this test in one of my recent commits, please check them.